src/main/resources/static/css/bootstrap.min.css

```diff
@@ -1,7 +1,7 @@
 /*!
- * Bootstrap v4.2.1 (https://getbootstrap.com/)
+ * Bootstrap v4.1.3 (https://getbootstrap.com/)
```

Comment:
Was this file down graded?

Reply:
Interesting... I thought I downloaded the last version....  I'll check that